lib/src/app/player.js

```diff
@@ -262,13 +262,13 @@ class CastPlayer {
           break;
 
         case OO.STATE.READY:
-          this.state = cast.receiver.media.PlayerState.IDLE;
+          this.state = this.params.autoPlay
```

Comment:
Instead of setting the state to what we expect it to be when auto-playing, is there a way for autoplay to play and have the play event set the state properly?

Reply:
I believe it's set properly. You just commenting first line of three lines ternary operator.